source/adapters/hip/usm.cpp

```diff
@@ -230,8 +230,12 @@ urUSMGetMemAllocInfo(ur_context_handle_t hContext, const void *pMem,
       }
       return ReturnValue(Pool);
     }
+    case UR_USM_ALLOC_INFO_SIZE: {
+      size_t RangeSize;
```

Comment:
Not sure if `pedantic` would complain about uninitialised variable here (probably yes), but you might want to give `RangeSize` a default value. Like we do in [here](https://github.com/oneapi-src/unified-runtime/blob/0959c61a557127baeec9cdcdc807969d647d84b9/source/adapters/hip/device.cpp#L167) for example.

Reply:
Good shout!